examples/reghd.py

```diff
@@ -0,0 +1,113 @@
+# The following two lines are only needed because of this repository organization
+import sys, os
+
+sys.path.insert(1, os.path.realpath(os.path.pardir))
+
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils.data as data
+
+# Note: this example requires the torchmetrics library: https://torchmetrics.readthedocs.io
+import torchmetrics
+from tqdm import tqdm
+
+from torchhd import functional
+from torchhd import embeddings
+from torchhd.datasets import AirfoilSelfNoise
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+print("Using {} device".format(device))
+
+DIMENSIONS = 10000  # number of hypervector dimensions
+NUM_FEATURES = 5  # number of features in dataset
+
+
+def transform(x):
```

Comment:
Can you move the two `transform` and `target_transform` functions to after the `TARGET_MEAN` definition? That way the order of the code is chronological and you're not using variables you're defining in the future